tools/bumper/component_commands.go

```diff
@@ -202,11 +202,10 @@ func (componentOps *gitComponent) getLatestTaggedFromBranch(repo, owner, branch,
 		return "", "", errors.Wrap(err, "Failed to get release tag refs from github client API")
 	}
 
-	branchCommits, _, err := componentOps.githubInterface.listCommits(owner, repo, &github.CommitsListOptions{ListOptions: github.ListOptions{PerPage: 100}, SHA: branch})
-	if err != nil {
-		return "", "", errors.Wrap(err, "Failed to get release tag refs from github client API")
-	}
-
+	const (
+		maxPagePaginate = 10
```

Comment:
if we use max, the bug can return if there are 1000 commits between
well i guess we can fix it once it happens if it does (rare)

Reply:
yes. We can't look forever, we can only look in the logical range..